frontend/src/scenes/trends/mathsLogic.tsx

```diff
@@ -236,12 +255,18 @@ export const mathsLogic = kea<mathsLogicType>({
             (mathDefinitions) => Object.entries(mathDefinitions).filter(([, item]) => item.type == PROPERTY_MATH_TYPE),
         ],
         mathDefinitions: [
-            (s) => [s.groupsMathDefinitions],
-            (groupOptions): Record<string, MathDefinition> => ({
-                ...BASE_MATH_DEFINITIONS,
-                ...groupOptions,
-                ...PROPERTY_MATH_DEFINITIONS,
-            }),
+            (s) => [s.groupsMathDefinitions, s.featureFlags],
+            (groupOptions, featureFlags): Record<string, MathDefinition> => {
+                const allMathOptions: Record<string, MathDefinition> = {
+                    ...BASE_MATH_DEFINITIONS,
+                    ...groupOptions,
+                    ...PROPERTY_MATH_DEFINITIONS,
+                }
+                if (!featureFlags[FEATURE_FLAGS.SESSION_ANALYSIS]) {
```

Comment:
I definitely agree - the issue that I was hitting is that `BASE_MATH_DEFINITIONS` has to include all `BasetMathType`s  to avoid typescript errors. So if we go the positive operation route, then we have to touch more stuff, and I was hoping to keep the feature flag changes consolidated (so we properly remove it when the time comes)